### DISCUSSION TITLE
GraphQL introspection on Appwrite Cloud?

### DISCUSSION THREAD

**Discussion by HerrMuellerluedenscheid** [appwrite/appwrite#7427] [Q&A] [answered]
### 💭 Description

I was trying to access the graphql introspection with postman. I set the `X-Appwrite-Key` and `X-Appwrite-Project` headers but received the warning:

    Could not load GraphQL schema.
    Received an invalid introspection response

Is introspection disabled even when being authentication? I probably missed that part of the documentation?

Cheers

**Comment by AdityaSrivastava185** [comment]
Make sure that even with authentication, your GraphQL server permits introspection. For security considerations, introspection might be turned off by default on some GraphQL servers. To enable introspection, check the server configuration or settings. If the problem still occurs, you can wish to look for any mistakes or anomalies in the introspection answer.

**Comment by HerrMuellerluedenscheid** [comment]
Thanks @AdityaSrivastava185 ! I meant the instance running at https://cloud.appwrite.io. Sorry, I wasn't clear there. Is there a way I can enable introspection on the cloud instance for a specific project?

**Comment by stnguyen90** [comment, marked as answer]
@HerrMuellerluedenscheid, introspection on Appwrite Cloud is not enabled.

**Reply by HerrMuellerluedenscheid** [reply]
Thanks! Worth mentioning this in the docs somewhere?
